tox.ini

```diff
@@ -1,9 +1,6 @@
 [tox]
 envlist = py35-django22,py38-django{22,30}
 
-[doc8]
-max-line-length = 120
```

Comment:
FYI: According to our documentation OEP, we no longer use hard line breaks in documentation.